config/application.rb

```diff
@@ -41,5 +41,15 @@ class Application < Rails::Application
 
     # Run background jobs asynchronously in an in-process thread pool.
     config.active_job.queue_adapter = :async
+
+    # Add error span to form fields with errors
+    config.action_view.field_error_proc = proc { |html_tag, instance|
```

Comment:
Whoa, TIL!

Is it possible to move the proc (or this entire section of the configuration) to a different file? I don't love the idea of this much "stuff" inside a configuration file.

Reply:
I agree, it's not the best spot. Would this be good for an initializer then?

Reply:
I've never added a proc to a configuration, but an initializer does sound like a good spot to start!

Reply:
Oh, I've looked at this before, and the `field_error_proc` is something... I'd looked at trying to make a PR to Rails to allow it to be configurable in form builders, but it's defined on `ActionView::Base`. I think it just has to live at the configuration level or in an initializer because of how Rails is written right now. It's like my open-source dream to find a way to disentangle this ancient piece of code into something more useful.